### PR TITLE
[minui] Package minui. Contributes to JB#39102

### PR DESCRIPTION
**Pull request by adenexter**

### CODE REVIEW

File: graphics.c [hunk 1]
```diff
@@ -51,6 +52,11 @@ static unsigned char gr_current_a = 255;
 
 static GRSurface* gr_draw = NULL;
 
+minui_backend* open_adf()
+{
+    return 0;
+}
```
Comment (jpetrell) [inline graphics.c:58]: Why?

**Comment by jpetrell** [comment]
LGTM

Packaging changes I guess should not be upstreamed, but if there are code fixes then those maybe.

File: graphics.c (hunk 1 above)
Comment (adenexter) [reply graphics.c:58]: We don't build graphics_adf.c.  Declaring an alternative symbol which returns null avoids the need to rewrite the initialization code to remove the use of it in turn reducing the potential for merge conflicts when merging in upstream changes.

**Comment by adenexter** [comment]
No code fixes.  The non build/packaging changes are replacing some string functions which don't exist in glibc, swapping a hardcoded image directory with something defined by the linking binary, and disabling the android framebuffer.

**Pull request merged**